images/win/scripts/Installers/Validate-Toolset.ps1

```diff
@@ -67,8 +67,8 @@ foreach($tool in $tools) {
 
     foreach ($version in $tool.versions) {
         # Add wildcard if missing
-        if (-not $version.Contains('*')) {
-            $version += '.*'
+        if ($version.Split(".").Length -lt 3) {

```

Comment:
What if we would like to use `x.*` pattern?

Reply:
It would still work, because all folders have x.x.x pattern (ex: 10.* it would change it to 10.*.*, so folder would be like 10.0.0).
We had to change old check for wildcard because it was adding extra * to versions (1.69.0 -> 1.69.0.*) and it was causing issues with validation part for boost.

Reply:
What if we would to use x.x without third number?

Reply:
It would still work: `10.2` -> `10.2.*`, `10` -> `10.*.*`, `10.2.2 ` -> `10.2.2`